io/net/virt-net/lpm.py

```diff
@@ -317,16 +318,18 @@ def test_migrate(self):
         '''
         Migrate the LPAR from given server to remote server.
         '''
-        cmd = self.form_virt_net_options()
-        self.do_migrate(self.server, self.remote_server, self.lpar, cmd)
+        if 'vnic' in self.options:
+            self.param = self.form_virt_net_options()
```

Comment:
`self.param` - local variable will be sufficient here.

Reply:
I had 2 same local method variable initialized in test_migrate and test_migrate_back fuctions.. but decided to have one single class variable to use in both method